### ISSUE TITLE
DHCP option sets with custom domain-names don't work

### ISSUE THREAD

**Issue by ajohnstone** [kubernetes/kops#1556]
DHCP option sets with custom domain-names don't work

```
kubectl get node ip-10-117-53-207.eu-west-1.compute.internal -oyaml
....
status:
  addresses:
  - address: 10.117.53.207
    type: InternalIP
  - address: 10.117.53.207
    type: LegacyHostIP
  - address: ip-10-117-53-207.ec2.internal
    type: Hostname
```

```
kops-1.5.0-alpha3 create cluster --kubernetes-version 1.5.2 --master-size m4.large --master-zones eu-west-1a,eu-west-1b,eu-west-1c --dns-zone Z1IJFNFB9OEVXQ --cloud aws --node-count 5 --node-size m4.large --zones eu-west-1b,eu-west-1c,eu-west-1a --state s3://k8s-kops-855439869752-cluster --network-cidr 10.117.0.0/16 --vpc vpc-c8f4d3ac --admin-access 0.0.0.0/0 --ssh-public-key ~/.ssh/photobox-eu.pub --topology=private --networking=kopeio-vxlan --bastion xxxx.photobox.com

```

@justinsb

**Comment by justinsb** [comment]
In this case, the dhcp options set had a domain-name of `ec2.internal`, though the default that EC2 creates would have been `eu-west-1.compute.internal`, and then the names would have matched.

I think we can fix this now in 1.5, because we can tell the apiserver to talk to the nodes by IP.

**Comment by ajohnstone** [comment]
changed dhcp option set to "domain-name = eu-west-1.compute.internal;domain-name-servers = AmazonProvidedDNS;" span new cluster up and working
